app/Services/WMURetriever.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Services;
+
+use App\Plot;
+use Illuminate\Support\Facades\Http;
+
+class WMURetriever
+{
+    /**
+     * @var \App\Plot
+     */
+    protected $plot;
+
+    /**
+     * Create the retriever.
+     *
+     * @return void
+     */
+    public function __construct(Plot $plot)
+    {
+        $this->plot = $plot;
+    }
+
+    /**
+     * Get the WMU for the plot.
+     *
+     * @return void
+     */
+    public function setWMU()
+    {
+        return rescue(function () {
```

Comment:
`@return` is `void` so no need to return anything.

```suggestion
       rescue(function () {
```